ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionProcessor.java

```diff
@@ -481,6 +481,7 @@ public TransactionProcessingResult processTransaction(
         }
       }
 
+      // TODO SLD are the log correct following EIP-7623?
```

Comment:
do we know enough to resolve this TODO?

Reply:
I didn't mean to commit that here but it's another issue that needs looking at ideally. I think we have all the info, need to make a call about how this log might be used versus the EIP-7623 spec changes